source/tyr/tests/integration/users_test.py

```diff
@@ -415,17 +416,20 @@ def test_update_invalid_user(mock_rabbit):
     assert status == 404
     assert mock_rabbit.call_count == 0
 
-def test_update_user(create_multiple_users, mock_rabbit):
+def test_update_user(create_multiple_users, mock_rabbit, geojson_feature):
     """
     we update a user
     """
-    user = {'login': 'user1', 'email': 'user1@example.com'}
+    user = {'login': 'user1', 'email': 'user1@example.com', 'shape': geojson_feature}
```

Comment:
+1 for the status
> And would be nice to test that updating shape with {} doesn't change the shape.

This will be achieved posting a json without shape attribute